pkgs/development/libraries/science/astronomy/indilib/indi-3rdparty.nix

```diff
@@ -28,27 +32,32 @@ stdenv.mkDerivation rec {
 
   inherit version src;
 
-  nativeBuildInputs = [ cmake ];
+  nativeBuildInputs = [ cmake pkg-config ];
 
   buildInputs = [
     indilib libnova curl cfitsio libusb1 zlib boost gsl gpsd
-    libjpeg libgphoto2 libraw libftdi1 libdc1394 ffmpeg fftw
+    libjpeg libgphoto2 libraw.dev libraw.lib libftdi1 libdc1394 ffmpeg fftw
+    limesuite libraw1394 udev
   ] ++ lib.optionals withFirmware [
     firmware
   ];
 
+  patches = [
+    ./atik.patch
+    ./playerone.patch
+  ];
+
   postPatch = ''
-    for f in indi-qsi/CMakeLists.txt \
-             indi-dsi/CMakeLists.txt \
-             indi-armadillo-platypus/CMakeLists.txt \
-             indi-orion-ssg3/CMakeLists.txt
-    do
+    for f in $(find . -name "CMakeLists.txt"); do
```

Comment:
This outputs many warnings for all those CMakeLists.txt files which do not need any substitution. Not sure if this is desirable or will confuse the next contributor.

Reply:
Can't we just list the relevant files here?

Reply:
Ah we could but it's a quite long list and easy to miss something: https://github.com/search?q=repo%3Aindilib%2Findi-3rdparty+%2Fudev%2F+language%3ACMake&type=code&p=1&l=CMake

I actually prefer the current approach :grimacing:

Reply:
The warnings are mostly harmless.

Nonetheless, it should be upstreamed.